tests/test_plots.py

```diff
@@ -0,0 +1,78 @@
+import scipy.stats as sps
+import numpy as np
+import unittest
+
+from GOFevaluation.utils import equiprobable_histogram
+from GOFevaluation.utils import plot_equiprobable_histogram
+
+
+class TestPlotEqualProbable(unittest.TestCase):
+    def test_plot_arguments(self):
+        n_data = 100
+        n_reference = 10000
```

Comment:
Since we only want to check if the plotting function throws an error we could go lower with the numbers (e.g. `n_data = 10`, `n_reference = 100` (the warning that will be thrown shouldn't be an issue I think), right?

Reply:
Actually, the calculation and plotting are super fast even with these high statistics. But I will change those `n`.